### ISSUE TITLE
Update docs to reflect login flow

### ISSUE THREAD

**Issue by claudiu-apetrei** [awkay/untangled-template#3]
Docs say any username/password and now it's:
```
(def valid-users
  (atom {1 {:uid 1 :name "Tony" :email "tony@nowhere.com" :password "letmein"}
         2 {:uid 2 :name "Joe" :email "joe@nowhere.com" :password "letmein"}}))
```

**Comment by awkay** [comment]
Oh, that would be a good fix :)

**Comment by awkay** [comment]
Fixed in 5d3203af93b89c112fa8766fcac6eb4d3374967a
